manifests/service.pp

```diff
@@ -27,16 +12,4 @@
       hasrestart => true,
     }
   )
-
-  if $mkhomedir and $manage_oddjobd {
-    ensure_resource('service', 'oddjobd',
-      {
-        ensure     => $service_ensure,
-        enable     => true,
-        hasstatus  => true,
-        hasrestart => true,
-        require    => Service[$sssd_service],
```

Comment:
I'm pretty sure oddjobd just needs to run and that what ever starts first of oddjobd or sssd is irrelevant. Authconfig changes on the other hand requires sssd to restart/reload.